hls4ml/backends/oneapi/oneapi_backend.py

```diff
@@ -154,6 +154,9 @@ def create_initial_config(self, part='Arria10', clock_period=5, io_type='io_para
             'WriteTar': write_tar,
         }
 
+        if 'use_bsp' in _:
```

Comment:
As I mentioned in #1218, this should be made an actual parameter, not taken from `_`. I am not clear how it is actually used, though.